framework/SupervisedLearning/SyntheticHistory.py

```diff
@@ -122,15 +125,15 @@ def __evaluateLocal__(self, featureVals):
     """
     pivots = self.pivotParameterValues
     result = np.zeros((self.pivotParameterValues.size, len(self.target) - 1)) # -1 is pivot
-    for algo in self.tsaAlgorithms:
+    for algo in self.tsaAlgorithms[::-1]:
```

Comment:
Maybe add comment here referencing skipping last ARMA comments in `__trainLocal__`?

Reply:
I don't think this is skipping any; it's just going through them in reverse order (as expected). 

This didn't come up before because we only were using one algorithm per SyntheticHistory ROM.

I double checked the test values, it does seem to be correctly generating both the ARMA and Fourier components. Let me know if that doesn't look true to you as well.

Reply:
I checked, and got the following:
```
>>> [1,2,3,4,5][::-1]
[5, 4, 3, 2, 1]
```

Reply:
You're right, I got it confused with `[1,2,3,4,5][:-1]`.